docs/pages/versions/unversioned/distribution/publishing-websites.md

```diff
@@ -170,3 +172,31 @@ Here are the formal instructions for deploying to GitHub Pages:
    - !! Your app is now available at the URL you set as `homepage` in your `package.json` (call your parents and show them! 😜)
 
    > When you publish code to `gh-pages`, it will create and push the code to a branch in your repo called `gh-pages`. This branch will have your built code but not your development source code.
+
+
+## Self hosting
+
+1. Build your Expo web app with `expo build:web`.
+
+2. Configure a webserver to host static files.
```

Comment:
Really appreciate you taking the time to write this up, but I think this is too vague right now. Even with the example below, I feel like it doesn't fit with the rest of the options on the page. If we do add documentation about self-hosting your own website, it should be thorough enough to take you from start-to-finish, like the rest of the options

Reply:
I added some explanations. Now, the 'self-hosting' section is less specific (Apache-only) but it's more straightforward to use.

Do you think it's enough? Should I add more details?

Reply:
I think this guide would have to go pretty deep. It's not easy for someone with little (or no) knowledge on publishing websites to read this and understand. They'd need to go through quite a bit of the apache docs

Reply:
I added additional explanations. In my opinion, adding more details is out-of-scope of Expo documentation: it's not about how to use `scp`.

If you're afraid people goes to Expo forum and wastes time of Expo support team because they have no clue about how to configure Apache, I can add something like:
```
Apache server, scp and sftp are not specific to Expo.
If you need more details about them, search on the web without 'Expo' term for better results.
```

What do you think about it?